tools/Migration/ExamplesMigration.ps1

```diff
@@ -16,14 +16,17 @@ function Start-Copy {
         [ValidateNotNullOrEmpty()]
         [string] $Module = "Users"
     )
+    
+    ../GenerateModules.ps1 -ModuleToGenerate "Teams"
```

Comment:
@timayabi2020, let's move the examples for the remaining modules in bulk, now that we are confident with the examples migration script. Moving the examples one module at a time will take a while.

The goal should be to have all examples migrated in a single v2.x preview release so we can close https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/1559.